src/index.html

```diff
@@ -1,14 +1,56 @@
-<!doctype html>
+<!DOCTYPE html>
 <html lang="en">
   <head>
+    <link rel="stylesheet">
     <meta charset="UTF-8">
     <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+      content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
     <link rel="stylesheet" href="./style.css">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="logo">
+        <a href="#home">
+          <img src="images/logo.png" alt="logo">
+        </a>
+      </div>
+      <nav class="nav">
+
+        <ul class="nav__list">
+          <li class="list__item">
+            <a href="#" class="nav__link  is-active">Apple</a>
+          </li>
+          <li class="list__item">
+            <a href="#" class="nav__link">Samsung</a>
+          </li>
+          <li class="list__item">
+            <a href="#" class="nav__link">Smartphones</a>
+          </li>
+          <li class="list__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="hover">Laptops&nbsp;&#38;&nbsp;computers
+            </a>
+          </li>
+          <li class="list__item">
+            <a href="#" class="nav__link">Gadgets</a>
+          </li>
+          <li class="list__item">
+            <a href="#" class="nav__link">Tablets</a>
+          </li>
+          <li class="list__item">
+            <a href="#" class="nav__link">Photo</a>
+          </li>
+          <li class="list__item">
+            <a href="#" class="nav__link">Video</a>
```

Comment:
```suggestion
            <a href="#" class="nav__link">Photo</a>
          </li>
          
          <li class="list__item">
            <a href="#" class="nav__link">Video</a>
```
you may add empty line here, between siblings